FoodBookApp/UI/Views/Settings/SettingsView.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  SettingsView.swift
+//  FoodBookApp
+//
+//  Created by Laura Restrepo on 11/05/24.
+//
+
+import Foundation
+import SwiftUI
+
+struct SettingsView: View {
+    @State private var daysSinceLastReview = UserDefaults.standard.bool(forKey: "sendDaysSinceLastReviewNotification")
+    @State private var days = UserDefaults.standard.integer(forKey: "daysSinceLastReview") - 1
+    @State private var reviewsUploaded = UserDefaults.standard.bool(forKey: "sendReviewsUploadedNotification")
+    @State private var lunchTime = UserDefaults.standard.bool(forKey: "sendLunchTimeNotification")
+    
+    let customGray2 = Color(red: 242/255, green: 242/255, blue: 247/255)
+    
+    var body: some View {
+        VStack(alignment: .leading) {
+            // Sheet rectangle
+            Rectangle()
+                .fill(Color.gray)
+                .frame(width: 40, height: 5)
+                .cornerRadius(3)
+                .padding(.top, 10)
+                .frame(maxWidth: .infinity, alignment: .center)
+            
+            // Title
+            Text("Settings")
```

Comment:
This can be defined in the swift UI sheet presentation i think, cause it would be cool if it could look like this:

![image](https://github.com/ISIS3510-202410-Team23/SwiftApp/assets/77981728/57823241-ad82-4398-b0f9-1b80596ef5f2)

Reply:
Sure, do you have any documentation I can check out?

Reply:
Maybe since this is going to be wrapped inside a `NavigationStack` for what I understood from @mar-cas3, maybe you could define the title there like we do with browse, for you or bookmarks. I could not find any documentation unfortunately, but i think this is what should happen ultimately.

Reply:
I couldn't find any doc either, and I once tried .navigationTitle which ultimately didn't do much difference. This is something that I did manually tho:

<img alt="settings sheet title" width="400" src="https://github.com/ISIS3510-202410-Team23/SwiftApp/assets/69609680/55a796e2-62fa-4b98-9217-2af0eb21f691">

Lmk what you think

Reply:
I think it looks great, but maybe we can have that as a placeholder until the proper navigation is implemented? Because maybe with what its being implemented it will add the title later, which will show two titles

Reply:
Okay, I'll leave it like this and once the nav w/ bug report is implemented we can check it out together

Reply:
Looking at the create review views, I just saw that CR1 has a manually set title, while CR2 has .navigationTitle, which makes me think that there will problably be no problem w/ leaving it like this and using .navigationTitle for the bug report view

Reply:
alrighty